### ISSUE TITLE
bug: firing too many non-essential crew members for a job

### ISSUE THREAD

**Issue by EndrosG** [endless-sky/endless-sky#2362]
Situation:
- 0 bunks free, 5 additional crew members on flag ship
- available job requires 2 bunks
- upon accept the game tells me I have to fire 4 (of those 5) guys...
- if I accept, the game really fires 4 people, i.e. I end up with 2 free bunks after the 2 passengers boarded

Version: 0.9.6 (steam)

* screenshot attached.
* job definition from savegame:
```
"available job" "Gold rush on Mars: Workers [0]"
	name "Workers to Mars"
	description "Transport this crew of 2 workers to Mars. Payment will be 18,000 credits."
	passengers 2
	job
	repeat 0
	to offer
		"Go to Mars: offered" != 0
		"event: Mars: gold rush ends" == 0
		random < 50
	destination Mars
	on complete
		dialog
			"You drop off the workers on the Mars and collect your payment of 18,000 credits."
		payment 18000
		"Mars: gold rush jobs" += 1

```
![endless_sky_v0_9_6__20170405__wrong_crew_adjustment_for_job](https://cloud.githubusercontent.com/assets/26883037/24721837/d421783e-1a41-11e7-845f-08277a38b089.jpg)

**Comment by MessyMix** [comment]
#2227

**Comment by endless-sky** [comment]
This should be fixed in 93721fe. What was happening is that you had hired extra crew on one of your escorts when it was your flagship, and the crew were being counted against your capacity internally but not being displayed.

**Comment by EndrosG** [comment]
just checked with my savegame and confirm that. I had changed flagship a few jumps earlier. The previous Hawk flagship still had a crew of 4 (required: 2).
Closing then.
